ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
https://docs.djangoproject.com/en/1.10/ref/migration-operations/#deletemodel :)
The only thing is indeed to never import models directly in migrations: not only because they could have been deleted, but also because they may evolve and then do not fit anymore in the migration being run.
I think it's better to always import models from the registry, which knows where in the history we are when applying this migration.